packages/ui/src/index.tsx

```diff
@@ -18,7 +18,7 @@ const uiConfig = JSON.parse(
   document.getElementById('__UI_CONFIG__')?.textContent || '{}'
 ) as UIConfig;
 
-initI18n({ lng: uiConfig.locale?.lng || 'en-US', basePath }).then(() => {
+initI18n({ lng: uiConfig.locale?.lng || navigator.language || 'en-US', basePath }).then(() => {
```

Comment:
What will happen if you change the language from settings & refresh the page?
It looks like we are not using the settings value again

Reply:
add a commit to S/L lang settings